packages/consent-manager/util/cookies.js

```diff
@@ -40,7 +42,14 @@ export function getDomain() {
 }
 
 export function loadPreferences() {
-  return cookies.getJSON(COOKIE_KEY)
+  const cookiesJSON = cookies.getJSON(COOKIE_KEY)
+  if (cookiesJSON) {
+    preferencesLoaded = true
+  } else if (preferencesLoaded) {
+    preferencesLoaded = false
+  }
```

Comment:
Question 🙋🏻 I'm not picturing when we would need to set `preferencesLoaded` to `false` if `preferencesLoaded` is already `true`. Is there a specific case where we need that to happen?

Reply:
@ashleemboyer good point. I added as a safeguard but yeah no scenarios that I can think of so it is probably unnecessary.